RELEASENOTES.docu

```diff
@@ -1681,4 +1681,12 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
   <build-id value="6132"><add-note> Unicode BiDi control characters
       are no longer permitted in DML source files, for security
       reasons. </add-note></build-id>
+  <build-id value="next"><add-note> Addressed code generation efficiency issues
+      when using array types in <tt>saved</tt> variables or parameters to
+      methods called with <tt>after</tt> <bug number="SIMICS-18440"/>.
+
+      Serialized representation of <tt>uint8[(be|le)_t]</tt> array types has
+      been changed. The old serialized representation is still accepted for
+      deserialization in order to preserve checkpoint compatibility.
```

Comment:
Is this paragraph warrented? Erik says it isn't necessary, but it's not harmful either.

Reply:
I don't think its necessary. Mentioning that the old rep is still valid but not what it is seems weird, and I dont want us to mention what the serialization type is (since we don't guarantee it).

Reply:
Conclusion is: doesn't matter.